GameServer/GameServer/GameLogic.cs

```diff
@@ -38,6 +49,7 @@ public static void Update()
 
         public static void initializeGame ()
         {
+            if (!gameStarted) return;
```

Comment:
You might only want to call initializeGame() if everyone is ready, instead of calling it and then checking if the game is started

Reply:
It's not called every time. That's just a "safety" check to make sure that the game has not already started when the method is called so the game is never re-initialized.